vulnerabilities/management/commands/import.py

```diff
@@ -71,44 +63,47 @@ def handle(self, *args, **options):
                 'Please provide at least one data source to import from or use "--all".'
             )
 
-        self.import_data(sources, options["cutoff_date"])
+        self.import_data(valid_sources(sources))
 
     def list_sources(self):
-        importers = IMPORTER_REGISTRY
+        importers = [importer.qualified_name for importer in IMPORTER_REGISTRY]
```

Comment:
this should just be:
```suggestion
        importers = list(IMPORTER_REGISTRY)
```

Reply:
Done